plugin/distributed/distributed.go

```diff
@@ -103,19 +103,23 @@ type OsqueryInt int
 // UnmarshalJSON marshals a json string that is convertable to an int, for
 // example "234" -> 234.
 func (oi *OsqueryInt) UnmarshalJSON(buff []byte) error {
-	// zero value
-	if string(buff) == `""` {
-		return nil
+	if len(buff) > 0 && buff[0] == '"' {
```

Comment:
This works but I feel like we can use `strconv.Unquote` if the string starts with quotes.